pysparkle/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
```

Comment:
Well, we can't do it without restructuring our repo, because the top level project cannot be built.
```
poetry build
Building stacks-azure-data-platform (0.1.0)
No file/folder found for package stacks-azure-data-platform
```

Reply:
Leaving as is - without restructuring the repo the top-level library cannot be packaged:
```
% poetry build
Building stacks-azure-data-platform (0.1.0)
No file/folder found for package stacks-azure-data-platform
```